awx/ui/src/api/models/Instances.js

```diff
@@ -25,6 +25,12 @@ class Instances extends Base {
   readInstanceGroup(instanceId) {
     return this.http.get(`${this.baseUrl}${instanceId}/instance_groups/`);
   }
+
+  removeInstance(instanceId) {
```

Comment:
The term `remove` is usually meant for `delete` actions when interfacing with REST. Since we are `POSTing` a field value, I would change this to `deprovisionInstance`